src/prefect/filesystems.py

```diff
@@ -908,7 +908,7 @@ async def get_directory(
 
             err_stream = io.StringIO()
             out_stream = io.StringIO()
-            process = await run_process(cmd, stream_output=(out_stream, err_stream))
+            process = await run_process([cmd], stream_output=(out_stream, err_stream))
```

Comment:
This will fix the windows version but I think it might break the non-windows variant. I _think_ what we want to do here instead is to build up `cmd` as a list from the beginning, ie: `cmd = ["git", "clone", self._create_repo_url()]` and then `cmd.append` whenever we need to add arguments.